app/controllers/course/assessment/question/programming_controller.rb

```diff
@@ -80,11 +80,12 @@ def generate
 
     generation_service = Course::Assessment::Question::CodaveriProblemGenerationService.new(
       @assessment,
-      params[:custom_prompt],
+      params,
+      # TODO: move these declarations (polyglot_language_name and _version) to polyglot repo
```

Comment:
https://github.com/Coursemology/coursemology2/pull/7631/commits/c830562bf693cf7869cbe352a7e8de463d3d3816#diff-ab3333bad9ce597da42d96a2c477e8d6ce7bb5ed856d3033101b9fa205159198L104

Don't add back this line

Reply:
I think this is actually other people making changes that deletes this, and the PR was made after I made this PR but got merged before this PR, and hence it seems like I added back this line.

Reply:
https://github.com/Coursemology/coursemology2/pull/7631/commits/c830562bf693cf7869cbe352a7e8de463d3d3816

Ya then don't add it back please.

Reply:
alright done. sorry for the trouble.
Since I moved this line, github was assuming this line as I'm changing this line, and hence mere rebasing didn't solve the issue; I have removed it manually and rearranged the commits accordingly